templates/NOTES.txt

```diff
@@ -7,9 +7,29 @@ Warning: `namespaceWhitelist` has been removed. Use multiple workspace providers
 {{- end }}
 
 {{- if .Values.dashboard }}
-Warning: The `dashboard` service will be removed in v1.21 (July 21st, 2021). You can safely delete the `dashboard` property.
+Warning: The `dashboard` service has been deprecated.
```

Comment:
```suggestion
Warning: The `dashboard` service is deprecated since Coder v1.20, and will be removed in v1.21.
```

Not sure why the removal comment was changed, that seems useful?